cmd/clusterctl/cmd/describe_cluster.go

```diff
@@ -161,20 +162,32 @@ func runDescribeCluster(name string) error {
 // printObjectTree prints the cluster status to stdout.
 func printObjectTree(tree *tree.ObjectTree) {
 	// Creates the output table
-	tbl := uitable.New()
-	tbl.Separator = "  "
-	tbl.AddRow("NAME", "READY", "SEVERITY", "REASON", "SINCE", "MESSAGE")
+	tbl := tablewriter.NewWriter(os.Stdout)
+	tbl.SetHeader([]string{"NAME", "READY", "SEVERITY", "REASON", "SINCE", "MESSAGE"})
+
+	tbl.SetAutoWrapText(false)
+	tbl.SetHeaderAlignment(tablewriter.ALIGN_LEFT)
+	tbl.SetAlignment(tablewriter.ALIGN_LEFT)
+
+	tbl.SetCenterSeparator("")
+	tbl.SetColumnSeparator("")
+	tbl.SetRowSeparator("")
+
+	tbl.SetHeaderLine(false)
+	tbl.SetBorder(false)
+	tbl.SetTablePadding("  ")
+	tbl.SetNoWhiteSpace(true)
```

Comment:
Can you rap all the formatting into a single function? It would be easier to maintain and the unit test can also reuse the same function.

Reply:
yes sure

Reply:
yes done